packages/jest-console/src/matchers.js

```diff
@@ -8,57 +8,72 @@ import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
  */
 import supportedMatchers from './supported-matchers';
 
+const createErrorMessage = ( spyInfo ) => {
+	const { spy, pass, calls, matcherName, methodName, expected } = spyInfo;
+	const hint = pass ? `.not${ matcherName }` : matcherName;
+	const message = pass
+		? `Expected mock function not to be called but it was called with:\n${ calls.map(
+				printReceived
+		  ) }`
+		: `Expected mock function to be called${
+				expected ? ` with:\n${ printExpected( expected ) }\n` : '.'
+		  }\nbut it was called with:\n${ calls.map( printReceived ) }`;
+
+	return () =>
+		`${ matcherHint( hint, spy.getMockName() ) }` +
+		'\n\n' +
+		message +
+		'\n\n' +
+		`console.${ methodName }() should not be used unless explicitly expected\n` +
+		'See file://gutenberg/packages/jest-console/README.md for details.';
```

Comment:
Should it be a publicly available URL instead? https://www.npmjs.com/package/jest-console

When using `@wordpress/jest-console` from npm, there is no `gutenberg/packages` in the path so it would be confusing.

Reply:
I also checked and found another npm package: https://www.npmjs.com/package/@wordpress/jest-console and package has a 'package/jest-console/README.md' file, which contains the same documentation offline in the package source. Which one would be better to use?

Reply:
The URL pointing to npm seems fine, thank you for applying the change 👍🏻